### ISSUE TITLE
Not evaluated macros in the upstream URL

### ISSUE THREAD

**Issue by FrostyX** [FrostyX/fedora-review-service#22]
See #14 

When the URL field contains macros, we don't evaluate them and set the Bugzilla URL with them. 

See for example https://bugzilla.redhat.com/show_bug.cgi?id=2163472 . 
The URL is set to `https://hackage.haskell.org/package/%{pkg_name}`

**Comment by juhp** [comment]
Thanks for noticing :-)

Also thanks for this service - it is fantastic!

**Comment by juhp** [comment]
Otherwise maybe don't set the url if it contains a macro?
Either way it would be good to do something about this

**Comment by FrostyX** [comment]
> Otherwise maybe don't set the url if it contains a macro?
> Either way it would be good to do something about this

I agree with you on both counts here @juhp. We use [Specfile](https://github.com/packit/specfile) in the code so evaluating the macro will be easy (is it safe though?) But in that case, ignoring the URL would be easy as well.

I'll try to prioritize the issue.
